templates/spec/rails_helper.rb

```diff
@@ -4,38 +4,23 @@
 require 'rspec/rails'
 require 'capybara/rspec'
 require 'capybara/rails'
+require 'selenium/webdriver'
 
-Capybara.register_driver :chrome do |app|
-  Capybara::Selenium::Driver.new(app, browser: :chrome)
-end
-
-Capybara.register_driver :headless_chrome do |app|
-  # The doc states, that the disable-gpu flag will some day not be necessary any more:
-  # https://developers.google.com/web/updates/2017/04/headless-chrome
-  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-    chromeOptions: { args: %w[headless disable-gpu] }
-  )
+ActiveRecord::Migration.maintain_test_schema!
 
-  Capybara::Selenium::Driver.new app, browser: :chrome, desired_capabilities: capabilities
-end
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
 
-Capybara.javascript_driver = :headless_chrome
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
 
-ActiveRecord::Migration.maintain_test_schema!
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
 
-RSpec.configure do |config|
-  config.before(:suite) do
-    DatabaseCleaner.strategy = :transaction
-    DatabaseCleaner.clean_with(:truncation)
+  config.before(:each, type: :system) do
+    driven_by :rack_test
   end
 
-  config.around(:each) do |example|
-    DatabaseCleaner.cleaning do
-      example.run
-    end
+  config.before(:each, type: :system, js: true) do
+    driven_by :selenium_chrome_headless
   end
-
-  config.include FactoryBot::Syntax::Methods
-
-  config.infer_spec_type_from_file_location!
 end
```

Comment:
It's kind of duplicated information. We have it in the rspec guide and here.

Reply:
Indeed, but I like to see, while following the setup guide, what I need to add, and at the same time a copy of the final version of the file as a reference.